miio/tests/test_fan.py

```diff
@@ -914,3 +915,170 @@ def delay_off_countdown():
 
         with pytest.raises(FanException):
             self.device.delay_off(-1)
+
+
+class DummyFanP9(DummyMiotDevice, FanP9):
```

Comment:
Move the tests also to `test_fan_miot.py`. The non-shared code between fans and miot fans should be kept separate.